ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/bootstrap/FactoryWorkspaceComponent.java

```diff
@@ -157,8 +158,8 @@ public void apply(PromiseError arg) throws OperationException {
      */
     private Promise<UsersWorkspaceDto> getWorkspaceToStart() {
         final WorkspaceConfigDto workspaceConfigDto = factory.getWorkspace();
-
-        switch (factory.getPolicies().getCreate()) {
+        final String create = factory.getPolicies() == null ? "perClick" : factory.getPolicies().getCreate();
```

Comment:
What about moving `"perClick"` to constant like `DEFAULT_POLICY`, + i think you should rename local variable to `policy` as well, as create looks a bit weird for me =)